modules/backend/assets/less/layout/fancylayout.less

```diff
@@ -1,3 +1,5 @@
+@background_color_1: #bf5d07;
```

Comment:
@ayumihamsaki Please remove this colour, as we should be using the variables from `modules/backend/assets/less/core/variables.less`.